x-pack/plugins/endpoint/server/routes/alerts/alerts.test.ts

```diff
@@ -24,6 +25,8 @@ describe('test alerts route', () => {
     mockClusterClient.asScoped.mockReturnValue(mockScopedClient);
     routerMock = httpServiceMock.createRouter();
     registerAlertRoutes(routerMock, {
+      indexPatternRetriever: new FakeIndexPatternRetriever('events-endpoint-*'),
```

Comment:
We can replace this with a jest mock. We can make a helper function that returns a mock that return the of the index pattern.

Reply:
Will do 👍